src/Native/Unix/System.Native/pal_console.c

```diff
@@ -51,7 +51,9 @@ static void WriteKeypadXmit() // used in a signal handler, must be signal-safe
     if (g_keypadXmit != NULL)
     {
         ssize_t ret;
+// clang-format off
         while (CheckInterrupted(ret = write(STDOUT_FILENO, g_keypadXmit, (size_t)(sizeof(char) * strlen(g_keypadXmit)))));
+// clang-format on
```

Comment:
Which format option is flagging this?  Can we instead remove it?  We use this pattern a lot, and the off/on comments just clutter things up.

Reply:
I have elaborated in details with a link to stackoverflow here: https://github.com/dotnet/corefx/pull/30495#issuecomment-398715808